packages/worker/src/api/routes/global/tests/realEmail.spec.ts

```diff
@@ -81,4 +93,19 @@ describe("/api/global/email", () => {
   it("should be able to send a password recovery email", async () => {
     await sendRealEmail(EmailTemplatePurpose.PASSWORD_RECOVERY)
   })
+
+  it("should be able to send an email with attachments", async () => {
+    tk.reset()
+    let bucket = "test-bucket"
+    let filename = "test.txt"
+    await objectStore.upload({
+      bucket,
+      filename,
+      body: Buffer.from("test data"),
+    })
+    tk.freeze(mocks.date.MOCK_DATE)
+    let presignedUrl = await objectStore.getPresignedUrl(bucket, filename, 600)
+
+    await sendRealEmail(EmailTemplatePurpose.WELCOME, [presignedUrl])
```

Comment:
As far as I can tell, nothing is asserting that the email that's sent contains the attachment we set. Could we create an assertion for that?

Reply:
Hm, had a look at this and I'm not sure the best way to go about it. Nodemailer doesn't return any info about attachments after sending, and we use ethereal to send a real email so I don't want to spy on and mock the function either.